sdks/python/apache_beam/runners/dataflow/internal/names.py

```diff
@@ -22,11 +22,9 @@
 
 from __future__ import absolute_import
 
-# TODO (altay): Move shared names to a common location.
 # Standard file names used for staging files.
 from builtins import object
 
-PICKLED_MAIN_SESSION_FILE = 'pickled_main_session'
 DATAFLOW_SDK_TARBALL_FILE = 'dataflow_python_sdk.tar'
 STAGED_PIPELINE_FILENAME = "pipeline.pb"
```

Comment:
STAGED_... constants could also move to a common place.

Reply:
I've moved `STAGED_PIPELINE_FILENAME ` to the common location. Though, it is only used in the Dataflow runner. Also, the `STAGED_PIPELINE_URL_METADATA_FIELD` constant was not used anywhere; I've removed it.